packages/amplify_auth_cognito/android/src/test/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AmplifyAuthCognitoPluginTest.kt

```diff
@@ -293,6 +293,28 @@ class AmplifyAuthCognitoPluginTest {
         verify(mockResult, times(1)).success(ArgumentMatchers.any<LinkedTreeMap<String, Any>>());
     }
 
+    @Test
+    fun getCurrentUser_returnsSuccess() {
+        // Arrange
+        doAnswer { invocation: InvocationOnMock ->
+            plugin.prepareUpdatePasswordResult(mockResult)
+            null as Void?
+        }.`when`(mockAuth).getCurrentUser()
+
+        val data: HashMap<*, *> = hashMapOf(
+                "username" to "username",
+                "userSub" to "userSub"
+        )
+        val arguments: HashMap<String, Any> = hashMapOf("data" to data)
+        val call = MethodCall("getCurrentUser", arguments)
+
+        // Act
+        plugin.onMethodCall(call, mockResult)
+
+        // Assert
+        verify(mockResult, times(1)).success(ArgumentMatchers.any<LinkedTreeMap<String, Any>>());
```

Comment:
`times(1)` is actually the default verification. Unless you add it for self-documenting, DAMP purposes, you do just:
```
verify(mock).method(...)
```

Oh also you don't need the trailing `;`, in Kotlin.

Reply:
Removed the comma.  I prefer to leave the explicit "1" and not rely on knowledge of defaults (for this particular project).